Examples/Nearest_Neighbour_Distance.ipynb

```diff
@@ -31,7 +31,9 @@
    },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    client = Client("tcp://10.6.48.33:8786")
</pre><p>Curious as to what this means/why we need it? Is it better than dong Client()? I get this error when running it:</p>

<pre class="ql-syntax" spellcheck="false">OSError: Timed out trying to connect to tcp://10.6.48.33:8786 after 30 s
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/363/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>